ruby-monday-blog/spec/features/member_can_navigate_to_post_from_index_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.feature "member can navigate to post from post index", type: :feature do
+  let!(:post) { Post.create(title: 'My Post', body: 'My Content') }
+
+  scenario "user visits post index and clicks on a post headline" do
+    # log_in_as user
+    visit posts_path
+    expect(page).to have_css "a", text: 'My Post'
```

Comment:
Apparently there is a built-in `have_link` method. The method signature is: `have_link('text inside link', options = {})`. In this case that would translate to: `expect(page).to have_link('My Post')`